### PR TITLE
Not found

### PR DESCRIPTION
**Pull request by kastrubait**
Add logo, page not found

### CODE REVIEW

File: src/app/base/components/vertical-menu/vertical-menu.component.scss [hunk 1]
```diff
@@ -10,21 +10,22 @@
   }
   .logo {
     position: absolute;
-    top: 1em;
-    left: 0.1em;
+    top: 0;
+    left: .2em;
```
Comment (Anntol) [inline src/app/base/components/vertical-menu/vertical-menu.component.scss:14]: please use 0.2 form

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -10,7 +11,7 @@ const routes: Routes = [
     loadChildren: () => import('src/app/base/base.module').then((m) => m.BaseModule)
   },
   { path: '', redirectTo: '/home', pathMatch: 'full' },
-  // { path: '**', component: PageNotFoundComponent },  // TODO Wildcard route for a 404 page
+  { path: '**', component: NotFoundPageComponent } // TODO Wildcard route for a 404 page
```
Comment (Anntol) [inline src/app/app-routing.module.ts]: TODO can be removed now

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -2,15 +2,16 @@ import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
 import { MainPageComponent } from './base/pages/main-page/main-page.component';
+// import { NotFoundPageComponent } from './base/components/not-found-page/not-found-page.component';
 
 const routes: Routes = [
   { path: 'home', component: MainPageComponent },
   {
-    path: 'page',
+    path: '',
```
Comment (coriander31415) [inline src/app/app-routing.module.ts]: `path: 'page'`

File: src/app/app-routing.module.ts [hunk 4]
```diff
@@ -2,15 +2,16 @@ import { NgModule } from '@angular/core';
...
     loadChildren: () => import('src/app/base/base.module').then((m) => m.BaseModule)
   },
   { path: '', redirectTo: '/home', pathMatch: 'full' },
```
Comment (coriander31415) [inline src/app/app-routing.module.ts:13]: `path: '**'`

File: src/assets/i18n/ua.json [hunk 5]
```diff
@@ -90,6 +90,10 @@
     "REG_NOTICE": "Будь ласка, зареєструйтесь, щоб побачити додаткову інформацію!",
     "PHONE_ERROR": "Будь ласка, введіть вірний номер телефону!"
   },
+  "PAGE": {
+    "NOT_FOUND": "Сторiнку не знайдено",
+    "GO_BACK": "Йдемо назад"
```
Comment (coriander31415) [inline src/assets/i18n/ua.json]: `"GO_BACK": "Повернутись"`

**Review by coriander31415 — APPROVED** [review]

File: src/app/base/components/project-list/project-list.component.scss [hunk 6]
```diff
@@ -119,14 +124,17 @@ $font-bold: 700;
...
-        color: $primary-text-color
-      }
+    }
+    .summ {
+      color: $primary-color-btn;
+    }
+    .funding {
+      text-align: right;
+      color: $secondary-text-color;
+      opacity: .7;
```
Comment (Anntol) [inline src/app/base/components/project-list/project-list.component.scss:134]: please use 0.7 form

**Pull request merged**